apps/admin-gui/src/app/users/pages/user-detail-page/user-accounts/user-accounts.component.html

```diff
@@ -0,0 +1,73 @@
+<h1 class="page-title">{{'USER_DETAIL.ACCOUNTS.TITLE' | translate}}</h1>
+<mat-spinner *ngIf="initLoading"></mat-spinner>
+<div class="d-flex">
+  <perun-web-apps-vo-search-select
+    *ngIf="!initLoading"
+    class="w-50"
+    [vos]="vos"
+    (voSelected)="loadMember($event)">
+
+  </perun-web-apps-vo-search-select>
+</div>
+<mat-tab-group *ngIf="!initLoading">
+  <mat-tab>
+    <ng-template matTabLabel>
+      {{'USER_DETAIL.ACCOUNTS.STATUS' | translate}}
+    </ng-template>
+    <ng-template matTabContent>
+      <mat-spinner *ngIf="loading"></mat-spinner>
+      <div *ngIf="!loading">
+        <div class="mt-4 mb-4">
+          <span class="subtitle mr-2">{{'USER_DETAIL.ACCOUNTS.MEMBER' | translate}}:</span>
+          <a class="member-link" [routerLink]="['/organizations', selectedVo.id, 'members', member.id]" queryParamsHandling="merge">
+            {{member.id}}
+          </a>
```

Comment:
![image](https://user-images.githubusercontent.com/32871167/128700523-6262a6ff-8a62-4b01-a308-4d557bf2dbe7.png)
just remove the space which is underlined  before the member id

Reply:
and after that merge it